library/attachment-viewer/src/main/java/im/vector/lib/attachmentviewer/SwipeToDismissHandler.kt

```diff
@@ -44,7 +44,7 @@ class SwipeToDismissHandler(
     @SuppressLint("ClickableViewAccessibility")
     override fun onTouch(v: View, event: MotionEvent): Boolean {
         when (event.action) {
-            MotionEvent.ACTION_DOWN                          -> {
+            MotionEvent.ACTION_DOWN -> {
```

Comment:
It just so happens that Kotlin files with multiple classes at root level don't get affected by when arrow alignment settings (and always autoformats) this way.

As we're making this change too, should we just leave it like this? @bmarty

Reply:
I am a bit bored with this rule about arrow alignment. Can we start by changing this one? i.e. not align arrows anymore? Can be a quick and small PR I think.

Reply:
Sounds good. Can do